src/Handler.ts

```diff
@@ -13,8 +13,8 @@ abstract class Handler {
     this.formatter = formatter;
   }
 
-  public handle(record: LogRecord): void {
-    const output = this.format(record);
+  public handle(record: LogRecord, format?: LogFormatter): void {
+    const output = format == null ? this.format(record) : format(record);
```

Comment:
Should this be `(format == null) ? ... : ...`? I thought the linter would change this.

Reply:
Also I prefer checking for positives first. Like `(format != null) ? format(record) : this.format(record)`. Also defaulting should usually be at the right side.

Reply:
Looks like the linting converts `(format != null)` to `format != null`. But otherwise I can flip it around.

Reply:
Yea please flip it around then resolve.